file_catalog/mongo.py

```diff
@@ -229,12 +321,14 @@ def append_distinct_elements_to_file(self, uuid, metadata):
 
         # update the file document
         update_query["$set"] = {"meta_modify_date": str(datetime.datetime.utcnow())}
-        result = self.client.files.update_one({'uuid': uuid}, update_query)
+        result = self.client.files.update_one({"uuid": uuid}, update_query)
 
         # log and/or throw if the update results are surprising
         if result.modified_count is None:
-            logger.warn('Cannot determine if document has been modified since `result.modified_count` has the value `None`. `result.matched_count` is %s' % result.matched_count)
+            logger.warning(
+                "Cannot determine if document has been modified since `result.modified_count` has the value `None`. `result.matched_count` is %s",
+                result.matched_count,
+            )
         elif result.modified_count != 1:
-            logger.warn('updated %s files with id %r',
-                        result.modified_count, uuid)
-            raise Exception('did not update')
+            logger.warning("updated %s files with id %r", result.modified_count, uuid)
+            raise Exception("did not update")
```

Comment:
`"did not update"` doesn't match so well with the logging statement above it.
Consider building a string and using it in both the logger warning and the exception message?

Reply:
you got it